client/vue.config.js

```diff
@@ -2,17 +2,28 @@ const fs = require('fs');
 const process = require('process');
 const path = require('path');
 const dotenv = require('dotenv');
+const webpack = require('webpack');
 
 const VuetifyLoaderPlugin = require('vuetify-loader/lib/plugin');
 
 // Read in .env file.
 const env = dotenv.parse(fs.readFileSync(path.resolve('..', '.env')));
 process.env.FLASK_SERVE_PORT = process.env.FLASK_SERVE_PORT || env.FLASK_SERVE_PORT || 5000;
 
+// Look for a git-sha file; if found, inject the value found in it into the
+// application.
+let GIT_SHA = null;
+if (fs.existsSync('git-sha')) {
+  GIT_SHA = JSON.stringify(fs.readFileSync('git-sha').toString());
```

Comment:
Why is `JSON.stringify` required here if the contents are already a string? I tried removing it locally but the whole site failed to load when I did.

Reply:
The way `DefinePlugin` works is by doing direct text replacement, so if GIT_SHA is meant to be a string, you need to define it here with its quotation marks. The cleanest way to do that is to call `JSON.stringify()` on it, since JSON strings come with quotation marks.